src/outputs/parthenon_hdf5.cpp

```diff
@@ -594,8 +596,8 @@ void PHDF5Output::WriteBlocksMetadata_(Mesh *pm, hid_t file, const HDF5::H5P &pl
 
   {
     // (LOC.)level, GID, LID, cnghost, gflag
-    hsize_t loc_cnt[2] = {num_blocks_local, 5};
-    hsize_t glob_cnt[2] = {max_blocks_global, 5};
+    hsize_t loc_cnt[2] = {num_blocks_local, 6};
```

Comment:
Maybe this should be a global const now since we had to change it.

Reply:
it is not obvious to me where the global should live though

Reply:
As far as I can tell this "structure" is intrinsically tied to restarts, so one "global" place I could imagine is the restart reader header.

Reply:
Added it there.